docs/mgmt/polishing.md

```diff
@@ -238,6 +239,30 @@ The content after `Serialized Name:` is the "original name" you would like to us
 
 Because this configuration adds quite a few extra content to the xml documents, it is not recommended to have this flag on an official SDK, be sure to remove the configuration and regenerate the SDK after all the rename work is done.
 
+### Example of the usage of `rename-mapping`
```

Comment:
I don't quite see much difference from this to the existing [Rename a property in class](https://github.com/Azure/autorest.csharp/blob/8aacacd88a75b84c2144ad2f538cfcd66ee5989b/docs/mgmt/polishing.md#rename-a-property-in-a-class). The example is in place already.
Is it possible to update the existing one with more details?

Reply:
The purpose of this is to include the **steps** because service team usually gets lost "what they should put as a key of rename mapping", they usually just put the name in the generated code and find it does not work then come to us and ask why.
Therefore I introduce this to include step by step tutorial

Reply:
Can we update the existing one with the new step-by-step instructions?